pages/about.js

```diff
@@ -18,10 +18,13 @@ const About = () => (
     <div className="row">
       <div className="col-xs-12 col-md-10 col-md-offset-1">
         <p>
-          OpenReview project is now a 501(c)(3) organization under the name OpenReview Foundation.
+          OpenReview project is now a 501(c)(3) organization under the name OpenReview
+          Foundation.
         </p>
         <p>
-          For inquiries, please contact <a href="mailto:hello@openreview-foundation.org">hello@openreview-foundation.org</a> or <a href="mailto:info@openreview.net">info@openreview.net</a>.
+          For inquiries, please contact{' '}
+          <a href="mailto:hello@openreview-foundation.org">hello@openreview-foundation.org</a>{' '}
+          or <a href="mailto:info@openreview.net">info@openreview.net</a>.
```

Comment:
Let's just keep info@openreview.net for now. I created a PR yesterday to remove `hello@openreview-foundation.org` [here](https://github.com/openreview/openreview-web/pull/2209)